README.md

```diff
@@ -32,47 +32,52 @@ A hotel booking application in React. Homework for the [CodeYourFuture React mod
 
-13. Extract the `<li>` containing "Pizzas" within the `<Restaurant />` component to a new component named `Order`. Move `orders` initial state set up in the `constructor` and the `addOrder` method from `<Restaurant />` to the new `<Order />` component. Make sure that clicking the "Add" button still increments the number of orders. Then replace the hard-coded string "Pizzas" in `<Order />` with a prop named `orderType`. Finally, render another `<Order />` component but this time with the prop `orderType="Salads"`.
+13. Extract the `<li>` containing "Pizzas" from the `<Restaurant />` component to a new component named `Order`. Also, move the declaration of the `orders` state and the `orderOne` function from the `<Restaurant />` component to the new `<Order />` component. Use the `<Order />` component in the `<ul>` list of the `<Restaurant />` component. Make sure that clicking on the "Add" button still increments the number of orders.
 
-14. Within the `<SearchResults />` component or it's child components, add an `onClick` handler to each row in the table (hint: on the `<tr>` element). When clicked, the row is "selected" and highlighted with a different colour. Hint: use state to add a class to the `className`. When clicking on the row for a second time, "unselect" the row and remove the coloured highlighting.
+14. Pass a new prop named `orderType` to the `<Order />` component with the value "Pizzas". Then render the `orderType` prop instead of "Pizzas" in the `<Order />` component. Make sure that "Pizzas" is still displayed on the screen. In the `<ul>` list of the `<Restaurant />` component, render 2 others `<Order />` components but this time pass different values for the `orderType` prop: "Salads" and "Chocolate cake". Verify that you can increment each number of items independently.
+
+15. Within the `<SearchResults />` component or its child components, add an `onClick` handler to each row in the table (hint: on the `<tr>` element). When clicked, the row is "selected" and highlighted with a different colour. When clicked again, the row is unselected and the coloured highlighting is removed. Hint: use a new state variable for each row to record if the row is selected or not, and use this value to set a class to the `className` prop of the row.
 
 ### Lesson 3
 
 15. Within your `<Header />` component, render the `<Clock />` component (that is provided for you in `src/Clock.js`). Fix the problem where the `setTimeout` timer is not **cleared** if the component is **unmounted**. Hint: look at the Clock exercise you did in class.
+    **TODO: is this question still relevant with React Hooks? Need to verify this is still in line with what's taught in the syllabus.**
```

Comment:
I believe this is still relevant with hooks. You would clear the timeout within the clean up of a useEffect. https://reactjs.org/docs/hooks-effect.html "Effects with Cleanup" section

Reply:
Not sure if this will be too complicated in the scope of the lesson. Lets discuss this..

Reply:
I have indeed added it to my PR: https://github.com/CodeYourFuture/syllabus/pull/417, but with an open question about whether it's worth including.

Reply:
Just noting that we decided to remove unmounting/cleaning up from the main syllabus and adding it as extra material. So I think these exercises should now be removed.